pages/dkp/kommander/2.2/dkp-upgrade/upgrade-konvoy/enterprise/index.md

```diff
@@ -0,0 +1,174 @@
+
+* Create an [on-demand backup][backup] of your current configuration with Velero.
+
+* Follow the steps listed in the [DKP upgrade overview][dkpup].
+
+* Ensure that all platform applications in the management cluster have been upgraded to avoid compatibility issues with the [Kubernetes version][releasenotes] included in this release. This is done automatically when [upgrading Kommander][upgradekomm], so ensure that you upgrade Kommander prior to upgrading Konvoy.
+
+* For air-gapped: Download the required bundles either at our [support site][supportsite] or by using the CLI.
+
+* For Azure, set the required [environment variables][envariables].
+
+* For AWS, set the required [environment variables][envariables2].
+
+* When upgrading, ensure that you set the flag `--etcd-version=3.4.13-0.`
+
+The following infrastructure environments are supported:
+
+* Amazon Web Services (AWS)
+
+* Microsoft Azure
+
+* Pre-provisioned environments
+
+## Overview
+
+To upgrade Konvoy for DKP Enterprise:
+
+1. Upgrade the Cluster API (CAPI) components
+1. Upgrade the core addons
+1. Upgrade the Kubernetes version
+
+Run all three steps on the management cluster (Kommander cluster) first. Then, run the second and third steps on additional managed clusters (Konvoy clusters), one cluster at a time using the KUBECONFIG configured for each cluster. If an AMI was specified when initially creating a cluster, you must build a new one with [Konvoy Image Builder][KIB] and pass it with `--ami.`
+
+<p class="message--note"><strong>NOTE:</strong> For pre-provisioned air-gapped environments, you must run <code>konvoy-image upload artifacts</code>.</p>
+
+For a full list of DKP Enterprise features, see [DKP Enterprise][dkpenterprise].
+
+<p class="message--note"><strong>NOTE:</strong> You must maintain your attached clusters manually. Review the documentation from your cloud provider for more information.</p>
+
+## Upgrade the CAPI components
+
+New versions of DKP come pre-bundled with newer versions of CAPI, newer versions of infrastructure providers, or new infrastructure providers. When using a new version of the DKP CLI, upgrade all of these components first.
+
+If you are running on more than one management cluster (Kommander cluster), you must upgrade the CAPI components on each of these clusters.
+
+<p class="message--warning"><strong>IMPORTANT:</strong>Ensure your <code>dkp</code> configuration references the management cluster where you want to run the upgrade by setting the <code>KUBECONFIG</code> environment variable <a href="https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/">to the appropriate kubeconfig file location</a>.</p>
+
+An alternative to initializing the KUBECONFIG environment variable is to use the <code>--kubeconfig=cluster_name.conf</code> flag.
+
+1. Run the following upgrade command for the CAPI components.
+
+```bash
+dkp upgrade capi-components
+```
+
+The output resembles the following:
+
+```bash
```

Comment:
Most--if not all--output blocks should use the "text" syntax, _not_ shell, because the output should not be highlighted as a shell script would. For example, "for" should not be highlighted blue here:
![image](https://user-images.githubusercontent.com/3445370/161829931-7383d479-2782-4908-a5f9-2465bdff9090.png)

```suggestion
```text
```

Reply:
This is good insight, thank you.

Reply:
Also, for future reference @mwrightd2 @dlipovetsky 

````
```bash
````

should be reserved for commands - because it keeps the copy/paste icon.

````
```sh
````

Will remove that in styling. So, `sh` would be a good option here, too.